packages/blocks/src/api/test/registration.js

```diff
@@ -106,12 +106,6 @@ describe( 'blocks', () => {
 			expect( block ).toBeUndefined();
 		} );
 
-		it( 'should reject blocks without a save function', () => {
```

Comment:
Alternatively (or even in place of the "filtered" case below), it could test that `save` was passed as a non-function.

Reply:
Right, I will add it as well.

Reply:
The reason why I used the approach with a filter was the fact that the existing check verifies if this is still an object after filtering. Maybe I should add another validation and error messaging to check that in test instead and update the existing test as you suggested.

Reply:
I figured out it's going to be easier to open a follow-up which improves error handling together with updates to unit tests: https://github.com/WordPress/gutenberg/pull/14529.